pkg/config/agent/config.go

```diff
@@ -267,6 +267,9 @@ type MulticastConfig struct {
 	// The interval for antrea-agent to send IGMP queries to Pods.
 	// Defaults to 125 seconds.
 	IGMPQueryInterval string `yaml:"igmpQueryInterval"`
+	// The versions of IGMP queries antrea-agent send to Pods.
+	// Defaults to v1, v2 and v3.
```

Comment:
```suggestion
	// Default is [1, 2, 3].
```

Reply:
Updated to `Defaults to [1, 2, 3].` I think defaults to is the correct wording because we have multiple uses of `defaults to.` In addition, Linux man pages have many `Defaults to,` https://man7.org/linux/man-pages/man7/arp.7.html, for instance.